src/stores/api/formatters/secrets.go

```diff
@@ -2,15 +2,15 @@ package formatters
 
 import (
 	"github.com/consensys/quorum-key-manager/src/stores/api/types"
-	"github.com/consensys/quorum-key-manager/src/stores/store/entities"
+	"github.com/consensys/quorum-key-manager/src/stores/entities"
 )
 
 func FormatSecretResponse(secret *entities.Secret) *types.SecretResponse {
 	return &types.SecretResponse{
 		ID:          secret.ID,
+		Version:     secret.Version,
 		Value:       secret.Value,
 		Tags:        secret.Tags,
-		Version:     secret.Metadata.Version,
```

Comment:
What about this metadata field, do we still need it?

Reply:
Actually it is needed, my mistake was to add "Version" as an attribute in the Secret entity which already have Metadata for it. I am gonna fix that now